content/intro-to-storybook/vue/es/simple-component.md

```diff
@@ -0,0 +1,277 @@
+
+```html
+<template>
+  <div class="list-item">
+    <input type="text" :readonly="true" :value="this.task.title" />
+  </div>
+</template>
+
+<script>
+  export default {
+    name: 'task',
+    props: {
+      task: {
+        type: Object,
+        required: true,
+        default: () => ({}),
+      },
+    },
+  };
+</script>
+```
+
+Arriba, renderizamos directamente `Task` basándonos en la estructura HTML existente de la app Todos.
+
+A continuación creamos los tres estados de prueba de Task dentro del archivo de historia:
+
+```javascript
+import { action } from '@storybook/addon-actions';
+import Task from './Task';
+export default {
+  title: 'Task',
+  // Our exports that end in "Data" are not stories.
+  excludeStories: /.*Data$/,
+};
+export const actionsData = {
+  onPinTask: action('onPinTask'),
+  onArchiveTask: action('onArchiveTask'),
+};
+
+export const taskData = {
+  id: '1',
+  title: 'Test Task',
+  state: 'Task_INBOX',
+  updated_at: new Date(2019, 0, 1, 9, 0),
+};
+
+const taskTemplate = `<task :task="task" @archiveTask="onArchiveTask" @pinTask="onPinTask"/>`;
+
+// default task state
+export const Default = () => ({
+  components: { Task },
+  template: taskTemplate,
+  props: {
+    task: {
+      default: taskData,
+    },
+  },
+  methods: actionsData,
+});
+// pinned task state
+export const Pinned = () => ({
+  components: { Task },
+  template: taskTemplate,
+  props: {
+    task: {
+      default: {
+        ...taskData,
+        state: 'TASK_PINNED',
+      },
+    },
+  },
+  methods: actionsData,
+});
+// archived task state
+export const Archived = () => ({
+  components: { Task },
+  template: taskTemplate,
+  props: {
+    task: {
+      default: {
+        ...taskData,
+        state: 'TASK_ARCHIVED',
+      },
+    },
+  },
+  methods: actionsData,
+});
+```
+
+Existen dos niveles básicos de organización en Storybook. El componente y sus historias hijas. Piensa en cada historia como una permutación posible del componente. Puedes tener tantas historias por componente como se necesite.
+
+- **Componente**
+  - Historia
+  - Historia
+  - Historia
+
+Para decirle a Storybook sobre el componente que estamos documentando, creamos una exportación `por defecto` que contiene:
+
+- `component` -- el componente en sí mismo,
+- `title` -- cómo referirse al componente en la barra lateral de la aplicación Storybook,
+- `excludeStories` -- información requerida por la historia, pero no debe ser presentada por la aplicación Storybook.
+
+Para definir nuestras historias, exportamos una función para cada uno de nuestros estados de prueba para generar una historia. La historia es una función que devuelve un elemento renderizado (es decir, una clase de componente con un conjunto de props) en un estado dado, exactamente como en Vue [Componente funcional sin estado](https://vuejs.org/v2/guide/render-function.html#Functional-Components).
```

Comment:
@IvanSotelo in this case the usage of the word class should be avoided. Probably go with "es decir, un componente con un conjunto de props", this to avoid any confusion that might ensue to either newcomers and seasoned developers